templates/scratch_org_expiry_email.txt

```diff
@@ -0,0 +1,5 @@
+Expiry Alert from MetaShare
+
+Your development scratch org for {{ repo_name }} {{ project_name }} on {{ task_name }} is set to expire in 3 days on {{ expiry_date }}. You have uncommitted changes on your org that will be deleted. If you do not want to lose these changes, please log into MetaShare as {{ user_name }}, navigate to the task, and click the Capture Task Changes button. We appreciate your contribution to the Salesforce community. Thanks!
+
+{{ metashare_login_link }}
```

Comment:
I'm not sure if it's just how it's being logged to console in development, but I'm getting escaped quotes in my email text in console. Also not a double-period after `p.m..`:

```
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: MetaShare Scratch Org Expiring with Uncommitted Changes
From: no-reply@metashare.org
To: jonny@oddbird.net
Date: Thu, 30 Jan 2020 15:37:14 -0000
Message-ID: <158039863419.4781.4545056238847125704@34cddd3de9cf>

Expiry Alert from MetaShare

Your development scratch org for MetaShare-Test Jonny&#x27;s Other Project on Jonny&#x27;s Task is set to expire in 3 days on Feb. 5, 2020, 8:55 p.m.. You have uncommitted changes on your org that will be deleted. If you do not want to lose these changes, please log into MetaShare as jgerigmeyer, navigate to the task, and click the Capture Task Changes button. We appreciate your contribution to the Salesforce community. Thanks!

https://example.com/accounts/github/login/

-------------------------------------------------------------------------------
```